### ISSUE TITLE
shield stealth d&d

### ISSUE THREAD

**Issue by SimonBpvp** [gamefreedomgit/Maelstrom#5427]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (http://wowhead.com/)
[//]: # (cata-twinhead.twinstar.cz)

**Description:**
When you (rogue) are in stealth and walk on a death and decay (dk) with a power word shield (priest) on you it breaks the stealth

**How to reproduce:**
Get a priest to shield you and then walk over a d&d

**How it should work:**
It shouldn't break the stealth

**Database links:**

**Comment by Renehadox** [comment]
> **Description:** When you (rogue) are in stealth and walk on a death and decay (dk) with a power word shield (priest) on you it breaks the stealth
> 
> **How to reproduce:** Get a priest to shield you and then walk over a d&d
> 
> **How it should work:** It shouldn't break the stealth
> 
> **Database links:**

retail proof?

**Comment by bonfax4** [comment]
Ground aoe should not break stealth through absorbs if it's already on the ground. Only the initial cast should if the rogue is within it, but walking over it should not. Death n decay is the only exception to this where the cast initial hit does not break stealth, but consecration and similar effects should if hit directly.

TLDR **Rogue stealth should not be broken by death n decay specifically unless the spell breaks the shield and damages the rogue directly in cata, changed in mop apparently**

Video proof : https://youtu.be/Aa8XYohz5wo?t=11

***star mention and hotfix of this issue, won't link it directly but i'ma quote since i think it's against the rules now (
you can easily find it by searching the spell ) : 

> As for that Stealth, it seems like it was changed in Mists of Pandaria. In Cataclysm, Death and Decay can't break Stealth with its cast, if you manage to hit a Rogue or Druid with absorption effect. Blizzard changed it a bit in Mists of Pandaria and that change allows it to break Stealth.

**Comment by bonfax4** [comment]
@Sketish  ^

**Comment by UltraNix** [comment]
Fixed.
